src/app/project-notifications/project-notifications-table-rows/project-notifications-table-rows.component.html

```diff
@@ -1,89 +1,100 @@
-        <p class="value"
-          *ngIf="rowData.decisionDate; else noDate">Notification Decision - {{ rowData.decision }} |
-          {{rowData.decisionDate?.toString().split('T')[0] || "-" }} </p>
-        <ng-template #noDate>
-          <p class="value">Notification Decision - {{ rowData.decision }}</p>
-        </ng-template>
-      </a>
-      <ng-template #noHyperlink>
-        <p class="value"
-          *ngIf="rowData.decisionDate; else noDate">Notification Decision - {{ rowData.decision }} |
-          {{rowData.decisionDate?.toString().split('T')[0] || "-" }} </p>
-        <ng-template #noDate>
-          <p class="value">Notification Decision - {{ rowData.decision }}</p>
-        </ng-template>
-      </ng-template>
-    </div>
-  </div>
-  <div class="row">
-    <div class="pn-info-block">
-      <label>Description:</label>
-      <p class="value"
-        [innerHTML]="(rowData.description || '-') | newlines"></p>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-sm-12 col-md-4 pn-info-block">
-      <label>Type</label>
-      <p class="value">{{rowData.type || "-" }}</p>
-    </div>
-    <div class="col-sm-12 col-md-4 pn-info-block">
-      <label>Location</label>
-      <p class="value">{{rowData.location || "-" }}</p>
-    </div>
-    <div class="col-sm-12 col-md-4 pn-info-block">
-      <label>Proponent</label>
-      <p class="value">{{rowData.proponent || "-" }}</p>
-    </div>
-    <div class="col-sm-12 col-md-4 pn-info-block">
-      <label>Sub-Type</label>
-      <p class="value">{{rowData.subType || "-" }}</p>
-    </div>
-    <div class="col-sm-12 col-md-4 pn-info-block">
-      <label>Region</label>
-      <p class="value">{{rowData.region || "-" }}</p>
-    </div>
-    <div class="col-sm-12 col-md-4 pn-info-block">
-      <label>Notification Trigger</label>
-      <p class="value">{{getTrigger(rowData) || "-" }}</p>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-sm-12 col-md-3">
-      <button *ngIf="rowData.documents && rowData.documents.length > 0"
-        class="btn btn-sm content-btn-alt"
-        type="button"
-        (click)="downloadDocuments(rowData)">
-        <span>View Documents</span>
-      </button>
-    </div>
-  </div>
+  class="col-12 pt-4 pn-location-info">
+  <mat-tab-group>
+    <mat-tab label="Project Notification Details">
+      <div class="tab-section mt-5 px-3">
+        <div class="row">
+          <div class="col-sm-12 col-md-6">
+            <h4>{{(rowData.name.toUpperCase() || '-')}}</h4>
+          </div>
+          <div class="col-sm-12 col-md-6">
+            <a *ngIf="rowData.commentPeriod?.daysRemainingCount > 0"
+              class="btn btn-primary view-pcp-btn"
+              title="View this Public Comment Period"
+              [routerLink]="['/pn', rowData._id, 'cp', rowData.commentPeriod?._id]">
+              <div class="pcp-indicator pcp-indicator-active"></div>
+              Open Comment Period - View/Submit
+            </a>
```

Comment:
Good candidate in the future for a component, not to be addressed here.

Reply:
1ccaec1